src/gitops/components/project/ProjectList.tsx

```diff
@@ -0,0 +1,568 @@
+
+  // Filter by search query if present (after other filters)
+  const filteredBySearch = React.useMemo(() => {
+    if (!searchQuery) return filteredData;
+
+    return filteredData.filter((project) => {
+      const name = project.metadata?.name || '';
+      const description = project.spec?.description || '';
+      const labels = project.metadata?.labels || {};
+
+      // Check if name, description, or labels match the search query
+      return (
+        name.toLowerCase().includes(searchQuery.toLowerCase()) ||
+        description.toLowerCase().includes(searchQuery.toLowerCase()) ||
+        Object.entries(labels).some(([key, value]) => {
+          const labelValue = value || '';
+          const labelSelector = `${key}=${labelValue}`;
+          const lowerSearchQuery = searchQuery.toLowerCase();
+          return (
+            labelSelector.toLowerCase().includes(lowerSearchQuery) ||
+            key.toLowerCase().includes(lowerSearchQuery)
+          );
+        })
+      );
+    });
+  }, [filteredData, searchQuery]);
+
+  const rows = useProjectsRowsDV(filteredBySearch, namespace, applications, appsLoaded);
+
+  // Check if there are projects initially (before search)
+  const hasProjects = React.useMemo(() => {
+    return sortedProjects.length > 0;
+  }, [sortedProjects]);
+
+  const getEmptyStateBody = () => {
+    if (searchQuery) {
+      return (
+        <>
+          {t('No Argo CD App Projects match the search filter')}{' '}
+          <strong>&quot;{searchQuery}&quot;</strong>.
+          <br />
+          {t('Try removing the filter or searching for a different term to see more App Projects.')}
+        </>
+      );
+    }
+    return namespace
+      ? t('There are no Argo CD App Projects in this project.')
+      : t('There are no Argo CD App Projects in all projects.');
+  };
+
+  const empty = (
+    <Tbody>
+      <Tr key="loading" ouiaId="table-tr-loading">
+        <Td colSpan={columnsDV.length}>
+          <EmptyState
+            headingLevel="h4"
+            icon={CubesIcon}
+            titleText={
+              searchQuery ? t('No matching Argo CD App Projects') : t('No Argo CD App Projects')
+            }
+          >
+            <EmptyStateBody>{getEmptyStateBody()}</EmptyStateBody>
+          </EmptyState>
+        </Td>
+      </Tr>
+    </Tbody>
+  );
+  const error = loadError && (
+    <Tbody>
+      <Tr key="loading" ouiaId={'table-tr-loading'}>
+        <Td colSpan={columnsDV.length}>
+          <ErrorState
+            titleText={t('Unable to load data')}
+            bodyText={t(
+              'There was an error retrieving App Projects. Check your connection and reload the page.',
+            )}
+          />
+        </Td>
+      </Tr>
+    </Tbody>
+  );
+  const isEmptyState = !loadError && rows.length === 0;
+
+  return (
+    <div>
+      {showTitle == undefined && (
+        <ListPageHeader
+          title={t('App Projects')}
```

Comment:
I believe this has to be the pluralized kind, eg. AppProjects, to be consistent with the other k8s resources list pages.